requirements/docs.txt

```diff
@@ -4,3 +4,4 @@ sphinx>=1.3,!=1.7.8
 numpydoc>=0.6
 sphinx-gallery
 scikit-learn
+dask[delayed]
```

Comment:
scikit-image depends on dask, so I don't think we need dask in docs requirements?

Reply:
I think delayed is different than array. Though they both depend on toolz, so I guess we satisfy the requirements for `dask[delayed]` today. 
https://github.com/dask/dask/blob/master/setup.py#L16